### DISCUSSION TITLE
PDF to PNG bad quality

### DISCUSSION THREAD

**Discussion by AlexksGud** [ImageMagick/ImageMagick#5762] [Help]
Converting PDF pages to images looks like screenshots with low quality and cut borders
`magick pd.pdf -gravity center p.png`

Zoomed Png:
![image](https://user-images.githubusercontent.com/58170415/202378809-50544000-68a1-48de-bbdd-7b1ff9caa2f3.png)
Zoomed Pdf:
![image](https://user-images.githubusercontent.com/58170415/202378971-d78a878e-5c1e-4d57-9d62-163010cdff20.png)

**Comment by Jmuccigr** [comment]
You have to set the density before you read in the pdf. For example assuming units are pixels per cm (you can set that too):

```
magick -density 160x160 pd.pdf -gravity center p.png
```

**Comment by oliverbrown00** [comment]


**Reply by snibgo** [reply]
Please don't post spam.
